### DISCUSSION TITLE
Allow holding left or right arrow to scroll continuously in the editor

### DISCUSSION THREAD

**Discussion by zyglrox** [Quaver/Quaver#4805] [Feature Request]
### Is your feature request related to a problem? Please describe.

Scrolling through the editor while charting using only your keyboard is currently inconvenient.

### Describe the solution you'd like

After holding left or right arrow for a brief period of time, start scrolling through the editor continuously until the key is released.

**Comment by EpiCreeper** [comment]
can we also add that to the song select menu? theres currently no real good way to navigate it with a keyboard
